data/index.html

```diff
@@ -95,6 +95,12 @@ <h2>Owie Status</h2>
                         <table class="cell-voltages" id="CELL_VOLTAGE_TABLE">%CELL_VOLTAGE_TABLE%</table>
                     </td>
                 </tr>
+                <tr>
+                    <th>Thermistor Temperatures</th>
```

Comment:
Call Battery Temps or something more user friendly. Longer strings overflow on mobile so keep that in mind.

Reply:
Changed to "Battery/BMS Temps" which is the same number of characters as the current longest string